fotoapparat/src/main/java/io/fotoapparat/hardware/Capabilities.java

```diff
@@ -60,6 +66,13 @@ public static Capabilities empty() {
     }
 
     /**
+     * @return list of supported preview scale types;
+     */
+    public Set<ScaleType> supportedPreviewScaleTypes() {
```

Comment:
Preview scale types are not capabilities of the camera. It is a configuration of the view itself, so it shouldn't be here.